### ISSUE TITLE
Stuck waiting for ansible-init to run on repeated runs

### ISSUE THREAD

**Issue by wtripp180901** [stackhpc/ansible-slurm-appliance#434]
If a run of the playbook fails and is rerun, it will get stuck on `TASK [Wait for ansible-init to finish]` presumably waiting for the full 20 minute timeout and requiring a reboot to fix

**Comment by sjpb** [comment]
I don't think this is an issue any more, have done multiple site.yml reruns and don't see this
